src/client/containers/chart-detail-page/chart-detailsmartpage-humidity.js

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react';
+import '../../components/detail-chart/detail-chart.style.css';
+import DetailChart from '../../components/detail-chart/detail-chart.component';
+import SidebarMenu from '../../components/side-navigation/sidebar.component';
+import ProgressBar from '../../components/progress-bar/progress-bar.component';
+import './chart-detail-smartpage.css';
+// import UpdateDateRange from '../../components/update-date-range/update-date-range.component';
+import ChartbarMenu from '../../components/chart-data-buttons/chartbar-buttons/chart-data-button.component';
+import { ChartDataContext } from './chart-detail-context';
+
+const value = [
```

Comment:
This should be named clearer. `customDataRanges`, `dataRangeChoices` or something like that... 🤷

Reply:
we will update the naming conventions 👍